graphql/quizzes/insertFirstProLang.ts

```diff
@@ -0,0 +1,13 @@
+import { gql } from "@apollo/client";
+
+export const INSERT_CODING_QUIZ_RESPONSE = gql`
+  mutation INSERT_CODING_QUIZ_RESPONSE($objects: [coding_quiz_insert_input!]!) {
```

Comment:
did you test that this mutation works inside of the hasura playground? or did you test that this works as expected some other way?

Reply:
Tested it in the Hasura playground!